interpreter/astwalker.go

```diff
@@ -152,11 +155,13 @@ func (w *astWalker) walkCall(node *exprpb.Expr) []Instruction {
 			evalCount += argGroupLens[i]
 			instructions = append(instructions, argGroup...)
 			if i != argCount-1 {
-				instructions = append(instructions,
-					NewJump(
-						argIDs[i],
-						instructionCount-evalCount,
-						jumpIfEqual(argIDs[i], types.Bool(function == operators.LogicalOr))))
+				if w.forceEval{
```

Comment:
Here and below, please consider phrasing this as though evaluating all branches is disabled by default.
```
if i != argCount - 1 && w.shortcircuit {
  instructions = append(instructions, NewJump(...))
}
```

Reply:
Done. Works better~ Thanks